modules/core/src/main/java/org/apache/ignite/internal/commandline/CommandHandler.java

```diff
@@ -421,40 +526,40 @@ private String requestDataFromConsole(String msg) {
 
     /** */
     private void printHelp() {
-        logger.log("Control.sh is used to execute admin commands on cluster or get common cluster info. " +
+        logger.info("Control.sh is used to execute admin commands on cluster or get common cluster info. " +
             "The command has the following syntax:");
-        logger.nl();
+        logger.info("");
 
-        logger.logWithIndent(CommandLogger.join(" ", CommandLogger.join(" ", UTILITY_NAME, CommandLogger.join(" ", getCommonOptions())),
+        logger.info(INDENT + CommandLogger.join(" ", CommandLogger.join(" ", UTILITY_NAME, CommandLogger.join(" ", getCommonOptions())),
             optional("command"), "<command_parameters>"));
-        logger.nl();
-        logger.nl();
+        logger.info("");
+        logger.info("");
 
-        logger.log("This utility can do the following commands:");
+        logger.info("This utility can do the following commands:");
 
         Arrays.stream(CommandList.values()).forEach(c -> c.command().printUsage(logger));
 
-        logger.log("By default commands affecting the cluster require interactive confirmation.");
-        logger.log("Use " + CMD_AUTO_CONFIRMATION + " option to disable it.");
-        logger.nl();
-
-        logger.log("Default values:");
-        logger.logWithIndent("HOST_OR_IP=" + DFLT_HOST, 2);
-        logger.logWithIndent("PORT=" + DFLT_PORT, 2);
-        logger.logWithIndent("PING_INTERVAL=" + DFLT_PING_INTERVAL, 2);
-        logger.logWithIndent("PING_TIMEOUT=" + DFLT_PING_TIMEOUT, 2);
-        logger.logWithIndent("SSL_PROTOCOL=" + SslContextFactory.DFLT_SSL_PROTOCOL, 2);
-        logger.logWithIndent("SSL_KEY_ALGORITHM=" + SslContextFactory.DFLT_KEY_ALGORITHM, 2);
-        logger.logWithIndent("KEYSTORE_TYPE=" + SslContextFactory.DFLT_STORE_TYPE, 2);
-        logger.logWithIndent("TRUSTSTORE_TYPE=" + SslContextFactory.DFLT_STORE_TYPE, 2);
-
-        logger.nl();
-
-        logger.log("Exit codes:");
-        logger.logWithIndent(EXIT_CODE_OK + " - successful execution.", 2);
-        logger.logWithIndent(EXIT_CODE_INVALID_ARGUMENTS + " - invalid arguments.", 2);
-        logger.logWithIndent(EXIT_CODE_CONNECTION_FAILED + " - connection failed.", 2);
-        logger.logWithIndent(ERR_AUTHENTICATION_FAILED + " - authentication failed.", 2);
-        logger.logWithIndent(EXIT_CODE_UNEXPECTED_ERROR + " - unexpected error.", 2);
+        logger.info("By default commands affecting the cluster require interactive confirmation.");
+        logger.info("Use " + CMD_AUTO_CONFIRMATION + " option to disable it.");
+        logger.info("");
+
```

Comment:
empty lines here can be removed

Reply:
I think empty lines here increase readability. I would prefer to keep them.